lib/core/macros.nim

```diff
@@ -339,11 +339,13 @@ proc quote*(bl: stmt, op = "``"): PNimrodNode {.magic: "QuoteAst", noSideEffect.
   ##       if not `ex`:
   ##         echo `info` & ": Check failed: " & `expString`
 
+from strutils import cmpIgnoreStyle, format
```

Comment:
A core module should not import anything. Use string concatenation instead of 'format' here.